src/App.css

```diff
@@ -1,5 +1,18 @@
+html, body, #root, #root > div {
+  height: 100%;
+  margin: 0px;
+  padding: 0px;
+}
+
+#root {
+  padding: 8px;
```

Comment:
This existed before as a browser margin default. This standardises it so all are the same.

Reply:
But we not always need the padding. For example, if this iframe is embedded in the app's settings, they are not necessary there.

Reply:
True, but for `scratchpad` that's fine as there isn't any config.